src/components/FloatingWidget/FloatingWidgetComponent.tsx

```diff
@@ -83,20 +83,23 @@ export class FloatingWidgetComponent extends React.Component<FloatingWidgetCompo
             <Rnd
                 className={className}
                 style={{zIndex: this.props.zIndex}}
-                default={{
-                    // Shift by 5 pixels to compensate for 5px CSS margins
-                    x: widgetConfig.defaultX !== undefined ? widgetConfig.defaultX : widgetsStore.defaultFloatingWidgetOffset + 5,
-                    y: widgetConfig.defaultY !== undefined ? widgetConfig.defaultY : widgetsStore.defaultFloatingWidgetOffset,
-                    width: widgetConfig.defaultWidth,
-                    height: widgetConfig.defaultHeight + headerHeight,
-                }}
+                size={{width: widgetConfig.defaultWidth, height: widgetConfig.defaultHeight + headerHeight}}
```

Comment:
are you sure we need to make this a controlled property? Can't we just simply use the updated values from the events when saving and restoring layouts?

Reply:
ah sorry use that prop without fully understanding it's meaning...a little bit confusion actually ToT

Reply:
adding update size/position in componentDidUpdate() fixes the problem with default prop